axe.d.ts

```diff
@@ -146,6 +153,17 @@ declare namespace axe {
 		}[];
 		cleanup?(callback: Function): void;
 	}
+	class VirtualNode {
```

Comment:
Shouldn't this be the abstract class instead?

Reply:
I thought we didn't expose the `VirtualNode` class and expected people to `class MyVNodeImpl extends axe.AbstractVirtualNode`. If so, we need to expose the class in the TS definition, otherwise the TS compiler won't know the abstract class exists.